packages/components/tsconfig.json

```
@@ -13,16 +13,11 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "noPropertyAccessFromIndexSignature": false,
+    "declarationDir": "declarations",
```

I believe these would need to be placed inside `dist` for an app to be able to access them
```suggestion
    "declarationDir": "./dist/declarations",
```

---

nevermind, `.npmignore` seems not to care about it, so it should end up being published